apps/omg_child_chain_rpc/lib/omg_child_chain_rpc/web/controllers/transaction.ex

```diff
@@ -30,5 +31,21 @@ defmodule OMG.ChildChainRPC.Web.Controller.Transaction do
       |> put_view(View.Transaction)
       |> render(:submit, result: details)
     end
+    |> increment_metrics_counter()
+  end
+
+  defp increment_metrics_counter(response) do
+    case response do
+      {:error, {:validation_error, _, _}} ->
+        Metrics.increment("transaction.failed.validation", 1)
+
+      %Plug.Conn{} ->
+        Metrics.increment("transaction.succeed", 1)
+
+      _ ->
+        Metrics.increment("transaction.fail.unidentified", 1)
```

Comment:
How about `transaction.failed.unidentified` to standardize with [`transaction.failed.validation`](https://github.com/omisego/elixir-omg/pull/775/files#diff-49f7310ecdb6873562b87df9745fdf12R40)?